Resolve failure to resolve types with RSG

Build: https://dev.azure.com/dnceng/public/_build/results?buildId=1034221&view=logs&j=80f594f0-dab9-53e4-691e-a6c036ec4279&t=04eab92f-dbd1-567e-0acd-be26c7e1c779&l=2426

```
##[error]D:\workspace\_work\1\s\src\Components\WebView\Samples\BlazorWpfApp\Microsoft.NET.Sdk.Razor.SourceGenerators\Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator\_Pages_Index.razor.cs(126,75): error CS0246: (NETCORE_ENGINEERING_TELEMETRY=Build) The type or namespace name 'AppState' could not be found (are you missing a using directive or an assembly reference?)

```

--- a/src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs
+++ b/src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs
@@ -390,7 +390,6 @@ namespace Templates.Test
 
         [Theory]
         [MemberData(nameof(TemplateData))]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/30851")]
         public Task BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_Works(TemplateInstance instance)
             => CreateBuildPublishAsync(instance.Name, args: instance.Arguments, targetFramework: "netstandard2.1");
 

